package.json

```diff
@@ -59,36 +59,39 @@
     "react-native": ">=0.50.0"
   },
   "dependencies": {
-    "css-select": "^4.2.1",
-    "css-tree": "^1.0.0-alpha.39"
+    "css-select": "^5.1.0",
+    "css-tree": "^1.1.3"
   },
   "devDependencies": {
+    "@babel/core": "^7.18.2",
```

Comment:
Could you briefly explain why all those newly added libs are necessary? I can see that `@babel/plugin-syntax-flow`, `@babel/plugin-transform-react-jsx` etc. were added. If there is a good reason behind it, I think it is probably ready to be merged then.

Reply:
Sure, good question.

After updating the `eslint` related packages that already exist, `yarn` shows warnings about peer deps:
```
warning " > eslint-plugin-flowtype@8.0.3" has unmet peer dependency "@babel/plugin-syntax-flow@^7.14.5".
warning " > eslint-plugin-flowtype@8.0.3" has unmet peer dependency "@babel/plugin-transform-react-jsx@^7.14.9".
warning " > babel-jest@25.5.1" has unmet peer dependency "@babel/core@^7.0.0".
```

So to resolve these warnings I've installed the peer deps as well using `yarn add -D`

Reply:
Ok, the last thing that I am afraid of is updating the `css-select` and `css-tree` deps since it will directly affect the users of the library. Is it fixing any vulnerabilities? If not, maybe it should be added in some other PR in order not to mix things for lib developers with those of lib users. Wdyt?

Reply:
Updating `css-select` is important to close vulnerability (this is the original reason I need to make this PR)

https://github.com/advisories/GHSA-rp65-9cf3-cjxr

The dep tree is:
`react-native-svg>css-select>nth-check`

Reply:
`css-tree` we could undo, but I was thinking if tests pass why not update it while I'm doing all this other work

Reply:
@WoLewicki please let me know your final decision here.

Reply:
Sorry for the delay, I think we can do it here, so it is ready to be merged :tada: Thanks for your contribution!